### DISCUSSION TITLE
Amazon Web Services thems

### DISCUSSION THREAD

**Discussion by BapediBoupi** [structurizr/themes#14] [Q&A] [answered]
### Question

Hello,

I see that high level components tags (available in 2020) has been removed since 2022 (for example: "Amazon Web Services - Cloud", "Amazon Web Services - Region"). What's the best practice / recommendation from Amazon to document this in 2023?

Do you plan to have a theme including 2023 + missing elements from previous versions or should I create my own?

Thanks!

**Comment by simonbrowndotje** [comment, marked as answer]
> What's the best practice / recommendation from Amazon to document this in 2023?

You would need to ask Amazon. 😀

> Do you plan to have a theme including 2023 + missing elements from previous versions or should I create my own?

I don't. The themes are based upon the icons available in the icon packs, and I don't know why Amazon chose to remove some of the icons in later versions. My suggestion would be to (1) create your own theme or (2) just use multiple themes. For example, if you're using the DSL:

```
theme https://static.structurizr.com/themes/amazon-web-services-2020.04.30/icons.json
theme https://static.structurizr.com/themes/amazon-web-services-2023.01.31/icons.json
```

**Reply by BapediBoupi** [reply]
Thanks! Using multiple themes is probably easier than maintaining my own :)
